### DISCUSSION TITLE
How to host on an external docker host

### DISCUSSION THREAD

**Discussion by djhutchie** [jetstreamapp/jetstream#1023] [Q&A]
As the title says, I am curious if i can set up a docker instance that will answer to the host rather than localhost? My first idea is to do a text replace of localhost to <intended host> before doing the ```docker build -t jetstream-app . comand. Is there a better way?

**Comment by paustint** [comment]
@djhutchie - the docker image should not be environment aware in any way, as this is just a runtime concern. So you don't need to do anything prior to building the image.

Assuming you are using docker-compose, then you should just need to adjust the host references here: `docker-compose.yml`

If you are not using docker compose, then however you are passing the required environment files to the run command is where you would need to make sure that the environment variables passed in have the correct host information.
